src/App.jsx

```diff
@@ -1,35 +1,44 @@
-import * as React from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import { useState, useEffect } from 'react'
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+import AddTodoForm from './AddTodoForm'
+import TodoList from './TodoList'
 
-const useSmiPersistantState = () => {
-  const [todoList, setTodoList] = React.useState(
-    JSON.parse(localStorage.getItem('savedTodoList')) || []);
-
-  React.useEffect (() => {
-  localStorage.setItem('savedTodoList', JSON.stringify(todoList))
-  }, [todoList])
+function useSemiPersistentState() {
+  const [todoList, setTodoList] = useState(JSON.parse(localStorage.getItem('savedTodoList')) || []);
+  useEffect(() => {
+    localStorage.setItem('savedTodoList', JSON.stringify(todoList));
+  }, [todoList]);
 
   return [todoList, setTodoList];
-}
+};
 
 function App() {
-  
-  const [todoList, setTodoList] = useSmiPersistantState();
+  const [todoList, setTodoList] = useSemiPersistentState();
 
   const addTodo = (newTodo) => {
     setTodoList([...todoList, newTodo]);
   };
+ 
+  const removeTodo = (id) => {
+  const updatedTodoList = todoList.filter(todo => todo.id !==id);
+  setTodoList(updatedTodoList);
+  };
 
   return (
     <>
+    <div>
       <h1>Todo List</h1>
-      
+    </div>
+    <div>
       <AddTodoForm onAddTodo={addTodo} />
-    
-      <TodoList todoList = {todoList} /> 
-    </>
+    </div>
+    <div>
+      <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+    </div>
+  </>
   );
-}
+};
```

Comment:
Remove this unnecessary semicolon.
```suggestion
}
```